components/tidb_query/src/rpn_expr/impl_cast.rs

```diff
@@ -661,6 +678,15 @@ mod tests {
         ft
     }
 
+    fn make_ret_field_type_7(parse_to_json: bool) -> FieldType {
```

Comment:
This function just be used in L1458, prefer to inline it to L1458 directly instead of defining a function.